Game/player.gd

```diff
@@ -1,25 +1,77 @@
 extends CharacterBody2D
 
-
 const SPEED = 300.0
 const JUMP_VELOCITY = -400.0
+var acceleration = 25
+
+var dash_speed = 700
+var is_dashing = false
+
+var can_jump = false
+
+var wall_jump_power = 500
```

Comment:
Could change these values to be exported just so they can be tweaked in the editor
```suggestion
@export var SPEED = 300.0
@export var JUMP_VELOCITY = -400.0
@export var acceleration = 25

@export var dash_speed = 700
var is_dashing = false

var can_jump = false

@export var wall_jump_power = 500
```